src/lib/collab/stack-client.js

```diff
@@ -383,5 +384,25 @@ export class ServiceClient {
       telepointerDoc
     )
   }
+
+  async postImage(name, noteId, file) {
+    return this.stackClient.fetchJSON(
+      'POST',
+      this.path(noteId, `images?Name=${name}`),
+      file,
+      { headers: { 'Content-Type': 'image/jpeg' } }
+    )
+  }
+
+  async getImages(noteId) {
+    return this.stackClient.fetchJSON(
+      'GET',
+      `/data/${appConstants.GET_IMAGES_DOCTYPE}/${noteId}/relationships/references`
+    )
+  }
+
+  async getImage(imageId) {
+    return this.stackClient.fetchJSON('GET', `/files/${imageId}`)
```

Comment:
Same. You should use CozyClient and not the stackClient. By using stackClient you bypass all the redux concept of CC and break ObservableQuery and else.

Reply:
same as https://github.com/cozy/cozy-notes/pull/196#discussion_r678116364

Reply:
Fixed